docs/Makefile

```diff
@@ -0,0 +1,12 @@
+all: build lint
+
+# Build the linter
+build:
+	cd linter && go build -o ./build/linter
```

Comment:
Cool that you added a linter. However, please move it to `misc/docs-linter`.

Keep this Makefile here.

Thank you.

Reply:
Would be nice to run it from the CI too.

Reply:
Moved the linter: [e7546dd](https://github.com/gnolang/gno/pull/2125/commits/e7546dd675bfc7db4cf3a2f38511b08cc84debc1)

Adding the linter to the CI seems like it is out of scope for this PR - let's leave it for another one.

Reply:
Added the linter to the CI: [8fad8c9](https://github.com/gnolang/gno/pull/2125/commits/8fad8c981841d4d63e0fd7ea475a5a4f91486e74)